portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -9,10 +9,10 @@
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
-
+    String msg = "Toya!"; 
```

Comment:
There's a couple extra spaces here:
```suggestion
  String msg = "Toya!"; 
```